### PR TITLE
[3] Successfully build consolidated package

### PR DESCRIPTION
**Pull request by anniehu4**
### Summary:
[ch187214]

Working on [consolidating to publish a single EDS package](https://docs.google.com/document/d/1JKgyVWybRlt-KnjQ1Y9c6m2eE9D-mv3q57Y070IK9mg/edit#)

This PR makes some cleanups to ensure the consolidated package successfully runs & builds

1. [consolidate `tokens`](https://github.com/chanzuckerberg/edu-design-system/pull/897)
2. [consolidate `components`](https://github.com/chanzuckerberg/edu-design-system/pull/898)
3. (this PR) replace yarn workspace commands
4. use `standard-version` instead of lerna in publish scripts
5. publish a new `@chanzuckerberg/eds` package!

### Test Plan:
- `yarn test` passes
- `yarn lint`
- `yarn start` & `yarn storybook:axe` work
- `yarn types` passes & story/spec files are still being checked locally:
Screen Shot 2022-03-16 at 5 10 24 PM
- `yarn build` shows expected output. in particular:
    - `index.d.ts` is in top level
  Screen Shot 2022-03-18 at 3 19 06 PM

    - each component folder has index, component, and css module files:
    Screen Shot 2022-03-18 at 3 19 15 PM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,154 @@
 
 ![Test CI](https://github.com/chanzuckerberg/edu-design-system/workflows/Test%20CI/badge.svg)
 
-The [Education Initiative](https://chanzuckerberg.com/education/)'s design system packages.
+The [Education Initiative](https://chanzuckerberg.com/education/)'s design system.
+
+## Setup
```
Comment (anniehu4) [inline README.md:7]: copied contents from `packages/components/README.md`

File: package.json [hunk 2]
```diff
@@ -44,16 +67,41 @@
       "webpack": false
     }
   ],
-  "author": "CZI",
-  "license": "MIT",
+  "peerDependencies": {
```
Comment (anniehu4) [inline package.json:70]: all merged in from `packages/components/package.json`

File: package.json [hunk 3]
```diff
@@ -1,20 +1,39 @@
 {
-  "name": "edu-design-system",
-  "private": true,
-  "description": "Learning Platform Design System",
-  "main": "index.js",
-  "workspaces": [
-    "packages/*"
+  "name": "@chanzuckerberg/eds",
```
Comment (ahuth) [inline package.json:2]: 😻

File: tsconfig.build.json [hunk 4]
```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "./tsconfig",
+  "include": ["src/index.ts", "css-modules.d.ts"],
```
Comment (anniehu4) [inline tsconfig.build.json:3]: the original config would output files to `lib/src/` (even with `baseUrl` option removed), which we don't want. This achieves the goal of:
- type checking all files, including stories & tests, during development
- but not including stories, tests, or files that are only used in stories/tests in the final build

Lmk if there are concerns or better ways to do this

**Review by anniehu4 — COMMENTED** [review]
Note that the build is working, but testing the publish / removing Lerna will be in another PR

File: package.json [hunk 5]
```diff
@@ -70,7 +70,7 @@
     "react": ">= 16.8.0"
   },
   "dependencies": {
-    "@headlessui/react": "^1.4.3",
```
Comment (anniehu4) [inline package.json]: hit the same unwanted snapshot changes issue as traject 🥲

File: package.json [hunk 6]
```diff
@@ -25,9 +44,13 @@
...
-    "test": "yarn workspaces foreach run test",
-    "types": "yarn workspaces foreach run types",
+    "start": "yarn build:tokens && yarn storybook",
+    "storybook": "start-storybook -p 6008",
+    "storybook:axe": "yarn run build:storybook && yarn run storybook:axeOnly",
+    "storybook:axeOnly": "axe-storybook --build-dir storybook-static",
+    "test": "jest",
+    "test:ci": "yarn run test --ci --coverage && cat ./coverage/lcov.info | codecov",
+    "types": "tsc --noEmit",
     "version": "./scripts/updateYarnLockForLerna.sh"
```
Comment (ahuth) [inline package.json:54]: I think this script and file can be deleted

**Review by ahuth — APPROVED** [review]
LGTM 🤘

File: package.json (hunk 6 above)
Comment (ahuth) [reply package.json:54]: Oh nvm, just saw your comment that Lerna would be removed in another PR.

File: tsconfig.build.json (hunk 4 above)
Comment (ahuth) [reply tsconfig.build.json:3]: Good thinking

Comment (anniehu4) [reply tsconfig.build.json:3]: aaaand this doesn't include any `Icons` 😅  will fix in the followup when I remove lerna, test publish, etc.

**Pull request merged**